packages/adapter/README.md

```diff
@@ -1,4 +1,4 @@
-# @node-escpos/core
+# @node-escpos/adapter
```

Comment:
What's this?

Reply:
And others.

Reply:
I forked from your main branch (it's behind your changeset branch), so they are your commits in the main branch.

Reply:
Oh, right.